src/ducks/transactions/TransactionRow/TransactionRowDesktop.jsx

```diff
@@ -79,9 +109,13 @@ const TransactionRowDesktop = props => {
       if (!ev.currentTarget.contains(ev.target)) {
         return
       }
-      showTransactionModal()
+      if (isSelectionModeActive) {
+        toggleTransactionSelection()
```

Comment:
I think it would be better if toggleTransactionSelection received the transaction id instead of having one toggle function per row. Anyway, we'll have n funcrtions because of the handleClickRow callback but still I think it makes it easier if there is only one toggle function.